src/engine/SCons/Node/PythonTests.py

```diff
@@ -110,6 +110,17 @@ def test___init__(self):
         vvv = SCons.Node.Python.Value('vvv')
         bi = SCons.Node.Python.ValueBuildInfo()
 
+
+class ValueChildTestCase(unittest.TestCase):
+    def test___init__(self):
+        """Test support for a Value() being an implicit dependency of a Node"""
+        value = SCons.Node.Python.Value('v')
+        node = SCons.Node.Node()
+        node._func_get_contents = 2  # Pretend to be a Dir.
+        node.add_to_implicit([value])
+        contents = node.get_contents()
+        assert len(contents) > 0
```

Comment:
Can you add also `assert contents == '%s %s'%(value.get_csig(), value.name)`

Reply:
Done, although the expected contents needs to have a \n at the end.